lib/vcr/cassette/serializers.rb

```diff
@@ -2,10 +2,11 @@ module VCR
   class Cassette
     # Keeps track of the cassette serializers in a hash-like object.
     class Serializers
-      autoload :YAML,  'vcr/cassette/serializers/yaml'
-      autoload :Syck,  'vcr/cassette/serializers/syck'
-      autoload :Psych, 'vcr/cassette/serializers/psych'
-      autoload :JSON,  'vcr/cassette/serializers/json'
+      autoload :YAML,       'vcr/cassette/serializers/yaml'
+      autoload :Syck,       'vcr/cassette/serializers/syck'
+      autoload :Psych,      'vcr/cassette/serializers/psych'
+      autoload :JSON,       'vcr/cassette/serializers/json'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.